app/views/devise/sessions/new.html.erb

```diff
@@ -0,0 +1,38 @@
+<br />
+<h2 class="headline">Log in</h2>
+
+<%= stylesheet_link_tag 'application', media: 'all' %>
+
+<div class="flash-messages">
+  <% flash.each do |key, value| %>
+    <div class="alert alert-<%= key %>">
+      <%= value %>
+    </div>
+  <% end %>
+</div>
+
+<%= form_for(resource, as: resource_name, url: session_path(resource_name)) do |f| %>
+  <br />
+  <div class="field">
+    <% f.label :email %><br />
+    <%= f.email_field :email, autofocus: true, autocomplete: "email", placeholder: "Email:", class: "form-control" %>
+  </div>
+  <br />
+  <div class="field">
+    <% f.label :password %><br />
+    <%= f.password_field :password, autocomplete: "current-password", placeholder: "Password:", class: "form-control" %>
+  </div>
+  <br />
+  <% if devise_mapping.rememberable? %>
+    <div class="field">
+      <%= f.check_box :remember_me %>
+      <%= f.label :remember_me %>
+    </div>
+  <% end %>
+  <br />
+  <div class="actions">
+    <%= f.submit "Log in", class: "submit-btn2" %>
+  </div>
+<% end %>
```

Comment:
- [ ] Unable to start a new session successfully after sign up and login. Let's ensure to improve upon this aspect.


**Sign in error:**
![image](https://github.com/jcmunav63/my-ror-blog/assets/12745474/d350392b-f6c8-40dc-9e52-5336c3f21ee8)